### ISSUE TITLE
DevTools default crashes stackblitz

### ISSUE THREAD

**Issue by warflash** [nuxt/starter#400]
Setting the default config for devtools to enabled in stackblitz prevents the app from working
```ts
export default defineNuxtConfig({
  devtools: { enabled: true },
});
```
https://stackblitz.com/edit/github-aktbgp?file=app.vue
-> Blocked a frame with origin "xx" from accessing a cross-origin frame.

**Comment by atinux** [comment]
cc @antfu

**Comment by danielroe** [comment]
I think we can go ahead and disable devtools in default stackblitz starter if there's not a way around this.

**Comment by antfu** [comment]
I will have a look into it, to see at least if we could make it non-blocking

**Comment by antfu** [comment]
This should be fixed with @nuxt/devtools@0.6.6, and DevTools is working in Stackblitz now.
